runtime.txt

```diff
@@ -0,0 +1 @@
+python-3.9.x
```

Comment:
Pinning to a Python minor version here so that CircleCI and our cloud.gov apps will be running code using the same (minor) Python version.

Chose to leave the patch flexible so that our cloud.gov environments could take advantage of higher patch version. There should be no breaking changes across patch (or even minor) versions according to the rules of [semver](https://semver.org/).

For more on specifying a Python version with cloudfoundry, see: 

https://docs.cloudfoundry.org/buildpacks/python/index.html#runtime